### DISCUSSION TITLE
should we write a tutorial like React router has exploring features of jotai

### DISCUSSION THREAD

**Discussion by chann44** [pmndrs/jotai#1593] [Q&A]
so React router has very extensive and good tutorial on there docs so can we do something like that so we can explore most of the things and build app while doing it 
i know there is a egghead video series on jotai i watched it's very fast paced and i struggled a bit so i want to ask can we do some tutorial section jotai where we build a small app that does some data fetching and other stuff while exploring the concepts and best practices of jotai atoms

**Comment by dai-shi** [comment]
Tutorials and best practice guides are something desired.
It's a good hint that I can make something based on the egghead course. (really sorry about the very fast paced videos. 😓 )
I will think about what I can do. Maybe some blog posts would be easier to start.
It would also be nice to add contents in our [docs](https://github.com/pmndrs/jotai/tree/main/docs).
Let us know if you are interested in working on something.
